malaria24/settings/base.py

```diff
@@ -73,6 +73,9 @@
     'djcelery',
 
     'raven.contrib.django.raven_compat',
+
+    'rest_framework',
+    'rest_framework.authtoken',
```

Comment:
Do these need to be added to the `requirements.txt`?

Reply:
@miltontony nope. I added it in a previous PR but forgot to add them here